es2015_exercise/readme.md

```diff
@@ -11,23 +11,42 @@ var person = {
         }.bind(this),1000)
     }
 }
+
+var person = {
```

Comment:
this can be simplified to:

```
let person = {
  fullName: "Harry Potter",
  sayHi() {
    setTimeout( () => console.log(`Your name is ${this.fullName}`), 1000);
  }
}
```